monitoring/remote_monitoring_demo/json_files/accelerator/profiles/perf/resource_optimisation_openshift.json

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
You can use this from the [manifests](https://github.com/kruize/autotune/blob/mvp_demo/manifests/autotune/performance-profiles/resource_optimization_openshift.json) itself